.TH "fcArrayIndexOutOfBoundException" 3 "2023年 一月 25日 星期三" "Version 00.01a07-dbg" "Freecraft" \" -*- nroff -*-
.ad l
.nh
.SH NAME
fcArrayIndexOutOfBoundException
.SH SYNOPSIS
.br
.PP
.PP
\fC#include <fcexceptions\&.hpp>\fP
.PP
继承自 \fBfcRuntimeException\fP \&.
.SS "Public 成员函数"

.in +1c
.ti -1c
.RI "\fBfcArrayIndexOutOfBoundException\fP (const char *whyException='the array index is out of bound\&.')"
.br
.in -1c

Public 成员函数 继承自 \fBfcRuntimeException\fP
.in +1c
.ti -1c
.RI "\fBfcRuntimeException\fP (const char *whyException='runtime exception occured')"
.br
.ti -1c
.RI "const char * \fBwhat\fP ()"
.br
.in -1c
.SH "详细描述"
.PP 
在文件 \fBfcexceptions\&.hpp\fP 第 \fB52\fP 行定义\&.
.SH "构造及析构函数说明"
.PP 
.SS "fcArrayIndexOutOfBoundException::fcArrayIndexOutOfBoundException (const char * whyException = \fC'the array index is out of bound\&.'\fP)\fC [inline]\fP"

.PP
在文件 \fBfcexceptions\&.hpp\fP 第 \fB55\fP 行定义\&..PP
.nf
55 : fcRuntimeException(whyException) {};
.fi


.SH "作者"
.PP 
由 Doyxgen 通过分析 Freecraft 的 源代码自动生成\&.
